examples/pytorch/audio-classification/run_audio_classification.py

```diff
@@ -391,6 +408,9 @@ def compute_metrics(eval_pred):
             checkpoint = last_checkpoint
         train_result = trainer.train(resume_from_checkpoint=checkpoint)
         trainer.save_model()
+        metrics['throughput'] = metrics['train_samples_per_second']
+        metrics['loss']= metrics['train_loss']
```

Comment:
It seems only logging after training is done..?

Reply:
That's right, the current code save the metric at the end of the training.

Reply:
Using the trainer can only log loss and lr each steps, throughput/train_samples_per_second is logged at the end of the training process. If those metrics are really need to be logged at each steps, please let me know and I will work on it.

Reply:
On research, I found that the trainer log loss, and lr each step at .state.log_history() not .metric(). However, the throughput is not logged though, I'll find a way to add log it each log interval